Sources/CredentialsFacebook/FacebookStructures.swift

```diff
@@ -0,0 +1,115 @@
+/// A structure representing the metadata provided by the Facebook API corresponding
+/// to a subject's profile picture. This includes the URL of the image and its width and height.
+/// If you wish to retrieve this information, include `let picture: FacebookPicture` in your
+/// user profile.
+public struct FacebookPicture: Codable {
+    public struct Properties: Codable {
```

Comment:
We are showing in the API doc as only 92% documented, at least partly because the properties within the structs in this file are not documented.  We should address this.